test/runner/allure_test.js

```diff
@@ -39,4 +43,34 @@ describe('CodeceptJS Allure Plugin', () => {
       done();
     });
   });
+
+  it('before suite errors are reported when running', (done) => {
```

Comment:
I would say `should report BeforeSuite errors`.

Reply:
https://github.com/Codeception/CodeceptJS/pull/2248/commits/69c6c960420ec3b32894e1c4bb6afe4cd7b4e23d